bindings/rust/evmc-vm/src/lib.rs

```diff
@@ -18,13 +18,19 @@ impl ExecutionResult {
         _status_code: ffi::evmc_status_code,
         _gas_left: i64,
         _output: Option<Vec<u8>>,
-        _create_address: ffi::evmc_address,
+        _create_address: Option<ffi::evmc_address>,
     ) -> Self {
         ExecutionResult {
             status_code: _status_code,
             gas_left: _gas_left,
             output: _output,
-            create_address: _create_address,
+            create_address: {
+                if let Some(_create_address) = _create_address {
```

Comment:
Could also do
```
if _create_address.is_some() {
    _create_address.unwrap()
} else {
```

Not sure which is nicer.

Reply:
i think it is fine as it is.